src/pages/msme/ViewCreditOptions.tsx

```diff
@@ -200,6 +204,14 @@ function ViewCreditOptions() {
                 options={SECTOR_TYPES}
                 placeholder={t('Sector')}
               />
+              <FormInput
+                className="w-3/5"
+                label={t('Annual Revenue')}
+                name="annual_revenue"
+                big={false}
+                type="currency"
+                placeholder={`${applicationContext.state.data?.borrower.currency}`}
```

Comment:
Now that this is simplified can we do:

```suggestion
                placeholder={applicationContext.state.data?.borrower.currency}
```